apps/web-giddh/src/app/accounting/journal-voucher/voucher/voucher.component.html

```diff
@@ -220,3 +295,9 @@ <h3 class="modal-title">Cheque details (if any)</h3>
 <aside-inventory-stock-group [autoFocus]="autoFocusStockGroupField" (closeAsideEvent)="closeCreateStock()"
     [class]="asideMenuStateForProductService" [@slideInOut]="asideMenuStateForProductService">
 </aside-inventory-stock-group>
+
+<!-- Recipt entry modal -->
+<ng-template #receiptEntry>
+    <receipt-entry [transaction]="currentTransaction" [activeCompany]="activeCompany" [voucherDate]="journalDate"
```

Comment:
Consider using `ngIf` with the component to show it only when `show()` method is called to avoid change detection.
```suggestion
    <ng-template #receiptEntry="bs-modal">
    <receipt-entry *ngIf="receiptEntry.isShown" [transaction]="currentTransaction" [activeCompany]="activeCompany" [voucherDate]="journalDate"
```